modules/nw-ingress-setting-select-subnet-LoadBalancerService.adoc

```diff
@@ -0,0 +1,144 @@
+// Modules included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-setting-select-subnet-LoadBalancerService_{context}"]
+= Choosing subnets while creating a LoadBalancerService ingress controller
+You can manually specify load balancer subnets for Ingress Controllers in an existing cluster. By default, load balancer subnets are automatically discovered by AWS, but specifying them in the Ingress Controller  overrides this, allowing for manual control.
+
+[NOTE]
+====
+This feature and procedure applies only to AWS. 
+====
+
+.Prerequisites
+* You must have an installed AWS cluster.
+* You must know the names or IDs of the subnets to which you intend to map your IngressController.
+
+.Procedure
+To create an Ingress Controller with manually specified load balancer subnets in {product-title} 4.17, you can follow these steps:
```

Comment:
It could be a bit confusing to hardcode the OpenShift version that it was released in, maybe customers think this only applies to 4.17? I think the version doesn't need to be declared here because all docs are organized by version, but I am not a docs expert so I could very well be wrong.